server/src/controllers/Events/resolver.ts

```diff
@@ -325,28 +324,25 @@ export class EventResolver {
           },
         },
       });
-    }
-
-    const rsvpName = getNameForNewRsvp(event);
-    const isSubscribedToEvent = userChapter ? userChapter.subscribed : true; // TODO add default event subscription setting override
-
-    const eventUserData: Prisma.event_usersCreateInput = {
-      user: { connect: { id: ctx.user.id } },
-      event: { connect: { id: eventId } },
-      rsvp: { connect: { name: rsvpName } },
-      event_role: { connect: { name: 'member' } },
-      subscribed: isSubscribedToEvent,
-    };
+    } else {
+      if (newRsvpName !== 'waitlist' && isSubscribedToChapter) {
+        await createReminder({
+          eventId,
+          remindAt: sub(event.start_at, { days: 1 }),
+          userId: ctx.user.id,
+        });
+      }
 
-    const eventUser = await prisma.event_users.create({
-      data: eventUserData,
-      include: eventUserIncludes,
-    });
-    if (rsvpName !== 'waitlist' && eventUserData.subscribed) {
-      await createReminder({
-        eventId: eventId,
-        remindAt: sub(event.start_at, { days: 1 }),
-        userId: ctx.user.id,
+      const eventUserData: Prisma.event_usersCreateInput = {
+        user: { connect: { id: ctx.user.id } },
+        event: { connect: { id: eventId } },
+        rsvp: { connect: { name: newRsvpName } },
+        event_role: { connect: { name: 'member' } },
+        subscribed: isSubscribedToChapter,
+      };
+      eventUser = await prisma.event_users.create({
+        data: eventUserData,
+        include: eventUserIncludes,
       });
```

Comment:
```suggestion
      });
      
      if (newRsvpName !== 'waitlist' && isSubscribedToChapter) {
        await createReminder({
          eventId,
          remindAt: sub(event.start_at, { days: 1 }),
          userId: ctx.user.id,
        });
      }
```
It needs to be moved after the `event_user` is created